modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/endpoint/ComposerRestService.java

```diff
@@ -633,7 +633,7 @@ public Response convertImage(@FormParam("sourceImage") String sourceImageXml, @F
 
     try {
       // Asynchronously convert the specified image
-      Job job = composerService.convertImage((Attachment) sourceImage, profileId);
+      Job job = composerService.convertImage((Attachment) sourceImage, StringUtils.split(profileId, ','));
```

Comment:
Same here: Proper serialization would be great

Reply:
This patch extend an always existing functionality. For this reason I would like to keep it backwards compatible. Changing it to an other format (json or xml) may break thinks.